root/etc/e-smith/events/actions/nethserver-antivirus-fix-rsync-perms

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+# 
+# This script is part of NethServer.
+# 
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+# 
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# skip event name
+
+shift
+
+# check if engine used is rsync
+
+backup=$1
+[ "$(/sbin/e-smith/db backups gettype "${backup}")" == "rsync" ] || exit 0
+
+# if rsync and if logs are included in the backup then fix their permissions
+
+if [ "$(/sbin/e-smith/config getprop backup-data IncludeLogs)" == "enabled" ]; then
+  if [[ -d "/var/log/clamav-unofficial-sigs/" ]]; then
+    /bin/chown -R clamupdate:clamupdate /var/log/clamav-unofficial-sigs
+  fi
+
+  if [[ -f "/var/log/freshclam.log" ]]; then
+    /bin/chown -R root:clamupdate /var/log/freshclam.log
```

Comment:
This is a single file, let's avoid `-R` option:
```suggestion
    /bin/chown root:clamupdate /var/log/freshclam.log
```

Reply:
Sure, thanks!